src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.ValueGenerator/Main.cs

```diff
@@ -110,7 +130,31 @@ public List<Result> Query(Query query)
             }
             catch (FormatException e)
             {
-               Log.Debug(GetTranslatedPluginTitle() + ": " + e.Message, GetType());
+                Log.Debug(GetTranslatedPluginTitle() + ": " + e.Message, GetType());
+                if (!string.IsNullOrEmpty(query.ActionKeyword))
+                {
+                    return GetSuggestionFuzzyResults(query, results);
+                }
+            }
+
+            return results;
+        }
+
+        private List<Result> GetSuggestionResults(Query query, List<Result> results)
+        {
+            foreach (var generatorItem in QueryHelper.GeneratorList)
+            {
+                results.Add(new Result
+                {
+                    Title = generatorItem.Key,
+                    SubTitle = generatorItem.Value,
+                    IcoPath = GetIcoPath(),
+                    Action = c =>
+                    {
+                        _context.API.ChangeQuery($"{query.ActionKeyword} {generatorItem.Key}");
+                        return false;
+                    },
+                });
```

Comment:
If the examples/descriptions are too long, we can add tool tips to make sure the user can always access the whole information.

Reply:
I just added the tooltip, but now the tooltip is always visible, is there any calculation to determine whether or not the text is too long? @htcfreek

Reply:
@azlkiniue 
This code `ToolTipVisibility = Visibility.Visible,` is the reason for the tool tip to be always visible. It overrides the automatic mouse hover detection.